src/main/java/org/cascadebot/cascadebot/CascadeBot.java

```diff
@@ -142,6 +152,20 @@ private void init() {
             return;
         }
 
+        if (Config.INS.getRedisHost() != null) {
+            redisClient = new Jedis(Config.INS.getRedisHost(), Config.INS.getRedisPort());
+            if (Config.INS.getRedisPassword() != null) {
+                redisClient.auth(Config.INS.getRedisPassword());
+            }
+            try {
+                redisClient.ping("Hello!");
+                LOGGER.info("Redis connected!");
+            } catch (JedisConnectionException e) {
+                LOGGER.warn("Failed to connect to redis", e);
```

Comment:
This should be an error message, not a warning

Reply:
I made it a warning as redis isn't strictly needed. the bot will run just fine without it, just message delete will not work. I can switch it over to an error if you wanted, but cause it wasn't needed I figured a warning was fine.

Reply:
I figure an error is appropriate since this will only be reached when Redis is configured in the config so the user intends to use it

Reply:
Fair enough.